src/java/org/apache/cassandra/config/ParameterizedClass.java

```diff
@@ -27,7 +28,7 @@
 import static org.apache.cassandra.utils.Shared.Scope.SIMULATION;
 
 @Shared(scope = SIMULATION)
-public class ParameterizedClass
+public class ParameterizedClass implements Serializable
```

Comment:
Only made this `Serializable` otherwise `nodetool getautorepairconfig` throws an exception, `token_range_splitter` doesn't currently show up in the output, will need to see how that can work.